code/modules/research/xenobiology/xenobiology.dm

```diff
@@ -581,20 +581,22 @@
 					H.AIStatus = AI_OFF
 					H.LoseTarget()
 				stopped_atoms |= M
+				var oldColor = M.color
+				M.color = "#704214"//sepia brown
```

Comment:
We don't use the color var anymore, look at elsewhere in the code where it is done, like the Ratvar immunity spell.